### ISSUE TITLE
No std lib autocomplete

### ISSUE THREAD

**Issue by zacaj** [zigtools/zls#575]
Trying to use the auto-installed ZLS through the VS Code extension on windows, but I can't get auto-completion to work for anything from the std lib.  Looking at the output logs, it seems to be correctly figuring out my zig lib path, but gives the error "Cannot resolve std library import, path is null." constantly

Full output up to the first error:

> Attempting to use zls @ c:\Users\zacaj\AppData\Roaming\Code\User\globalStorage\augusterame.zls-vscode\zls_install\zls.exe
> info(main): No config file zls.json found.
> info(config): Using zig executable A:\root\zig-windows-x86_64-0.10.0-dev.3442+fd3415ad5\zig.exe
> info(config): Using zig lib path 'A:\root\zig-windows-x86_64-0.10.0-dev.3442+fd3415ad5\lib'
> [Info  - 4:24:04 PM] [info-Server] zls initialized
> [Info  - 4:24:04 PM] [info-Server] Server.ClientCapabilities{ .supports_snippets = true, .supports_semantic_tokens = true, .supports_inlay_hints = true, .hover_supports_md = true, .completion_doc_supports_md = true, .label_details_support = true, .supports_configuration = true }
> [Info  - 4:24:04 PM] [info-Server] Using offset encoding: utf16
> [Info  - 4:24:04 PM] [info-Server] Requesting configuration!
> debug(doc_store): Opened document: file:///a%3A/source/test%20zig/src/main.zig
> debug(doc_store): Opened document: file:///a%3A/source/test zig/build.zig
> debug(doc_store): Document is a build file, extracting packages...
> debug(doc_store): Finished zig run for build file file:///a%3A/source/test zig/build.zig
> debug(doc_store): Cannot resolve std library import, path is null.

It looks like this error is coming from this function in the code (https://github.com/zigtools/zls/blob/ea24928002c4ee0c90050449c091e221aed36f72/src/DocumentStore.zig#L744), but there doesn't seem to be much that can go wrong there, especially considering that the zig lib is being logged correctly and I'm not getting any "Failed to resolve zig std library path"

**Comment by SuperAuguste** [comment]
Can you please run the `Install Server` command in the VSCode command palette and restart the editor? Just merged a fix so that set of actions should fix this :)

**Comment by zacaj** [comment]
Working, now, thanks!

Note, I first just tried running Install Server, and it didn't work.  Then I tried to do a server restart, didn't work.  Reloaded VS Code, didn't work.  I checked the server install folder, and the date modified on the exe hadn't changed.  I deleted the whole zls_install folder, then ran Install Server, and that fixed it.  Not sure if that was just some weird thing on my end or not...
